Tests/Routing/Loader/RestYamlCollectionLoaderTest.php

```diff
@@ -219,7 +219,7 @@ public function testControllerAsServiceWithClassName()
 
         // We check that it's "controller:method" if sf < 4.1 (controller as service) and not "controller::method"
         $this->assertEquals(
-            UsersController::class.(!method_exists(Kernel::class, 'getAnnotatedClassesToCompile') ? ':' : '::').'getUsersAction',
+            UsersController::class.(Kernel::VERSION_ID >= 40100 ? '::' : ':').'getUsersAction',
```

Comment:
Is this actually depending on the version of the HttpKernel component? It depends on the Routing component being 4.1 or higher, doesn't it?

Reply:
It depends on the httpkernel component because the `ContainerControllerResolver` belongs to the component :slightly_smiling_face: